strerr_sys.c

```diff
@@ -1,7 +1,7 @@
 #include "error.h"
 #include "strerr.h"
 
-struct strerr strerr_sys;
+struct strerr strerr_sys = {0,0,0,0};
```

Comment:
Please also add a comment here and say something like "workaround for MacOS linker bug" so one does not need to dig project history to see what this is for.

Reply:
Reasonable, done.